package.json

```diff
@@ -50,7 +51,23 @@
     ]
   },
   "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-prettier": "^6.12.0",
     "eslint-loader": "^4.0.2",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.21.3",
+    "eslint-plugin-react-hooks": "^4.1.2",
+    "husky": "^4.3.0",
+    "lint-staged": "^10.4.0",
+    "prettier": "^2.1.2",
     "webpack-cli": "^3.3.11"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix"
```

Comment:
i believe this is missing commands to prettify as well. Here is a project that has prettier/linting already working

https://github.com/starter-code/res-gen/blob/master/package.json#L66-L71

Reply:
I think this will work with just eslint because it has the prettier plugin (it's how I usually set it up) - I will attach a video below that shows it should be working

Reply:
interesting. the prettier/eslint commit worked for me